packages/web-components/src/components/rux-modal/rux-modal.tsx

```diff
@@ -103,7 +123,10 @@ export class RuxModal {
 
         if (buttonSet.length > 0) {
             const defaultButton = buttonSet[buttonSet.length - 1]
-            return defaultButton
+            //* Need to get the native <button> to focus. This isn't necessary if we add the delegateFocus option to the stencil shadow in Stencil 2.10
+            const shadow = defaultButton.shadowRoot?.querySelector('button')
```

Comment:
Focusing the actual native button in the shadow DOM. Stencil 2.10 allows for a `delegateFocus: true` option that would handle this for us in the future.